<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
    <line>
      <![CDATA[Stetting up report and Test is getting ready]]>
    </line>
    <line>
      <![CDATA[Setting done-Test can be started]]>
    </line>
    <line>
      <![CDATA[Browser and Application are up and running]]>
    </line>
  </reporter-output>
  <suite name="Suite" duration-ms="2728" started-at="2020-02-12T14:34:49Z" finished-at="2020-02-12T14:34:52Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="2728" started-at="2020-02-12T14:34:49Z" finished-at="2020-02-12T14:34:52Z">
      <class name="com.learnautomation.testcases.LoginTestHRM">
        <test-method status="PASS" signature="setUpSuit()[pri:0, instance:com.learnautomation.testcases.LoginTestHRM@291ae]" name="setUpSuit" is-config="true" duration-ms="631" started-at="2020-02-12T14:34:49Z" finished-at="2020-02-12T14:34:49Z">
          <reporter-output>
            <line>
              <![CDATA[Stetting up report and Test is getting ready]]>
            </line>
            <line>
              <![CDATA[Setting done-Test can be started]]>
            </line>
          </reporter-output>
        </test-method> <!-- setUpSuit -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.learnautomation.testcases.LoginTestHRM@291ae]" name="setup" is-config="true" duration-ms="2622" started-at="2020-02-12T14:34:49Z" finished-at="2020-02-12T14:34:52Z">
          <reporter-output>
            <line>
              <![CDATA[Browser and Application are up and running]]>
            </line>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="loginToApp()[pri:1, instance:com.learnautomation.testcases.LoginTestHRM@291ae]" name="loginToApp" duration-ms="70" started-at="2020-02-12T14:34:52Z" finished-at="2020-02-12T14:34:52Z">
          <exception class="org.openqa.selenium.WebDriverException">
            <message>
              <![CDATA[Returned value cannot be converted to WebElement: {stacktrace=Backtrace:
Ordinal0 [0x008FA903+1550595]
Ordinal0 [0x0087A701+1025793]
Ordinal0 [0x007FC58E+509326]
Ordinal0 [0x00790FB9+69561]
Ordinal0 [0x007A8DD7+167383]
Ordinal0 [0x007A77F2+161778]
Ordinal0 [0x007A73CA+160714]
Ordinal0 [0x0078221E+8734]
Ordinal0 [0x00782606+9734]
Ordinal0 [0x00782C80+11392]
Ordinal0 [0x008942C7+1131207]
GetHandleVerifier [0x009970FD+523789]
GetHandleVerifier [0x00996E90+523168]
GetHandleVerifier [0x0099E1E7+552695]
GetHandleVerifier [0x009978FA+525834]
Ordinal0 [0x0088B7FC+1095676]
Ordinal0 [0x00782053+8275]
Ordinal0 [0x00781DA7+7591]
GetHandleVerifier [0x00CFA3BC+4075724]
BaseThreadInitThunk [0x74C86359+25]
RtlGetAppContainerNamedObjectPath [0x76F07B74+228]
RtlGetAppContainerNamedObjectPath [0x76F07B44+180]
, error=invalid session id, message=invalid session id}
Build info: version: '3.1.0', revision: 'b94c902', time: '2017-02-16 12:21:31 -0800'
System info: host: 'MININT-SMKHDAE', ip: '192.168.58.87', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_221'
Driver info: driver.version: RemoteWebDriver]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.WebDriverException: Returned value cannot be converted to WebElement: {stacktrace=Backtrace:
Ordinal0 [0x008FA903+1550595]
Ordinal0 [0x0087A701+1025793]
Ordinal0 [0x007FC58E+509326]
Ordinal0 [0x00790FB9+69561]
Ordinal0 [0x007A8DD7+167383]
Ordinal0 [0x007A77F2+161778]
Ordinal0 [0x007A73CA+160714]
Ordinal0 [0x0078221E+8734]
Ordinal0 [0x00782606+9734]
Ordinal0 [0x00782C80+11392]
Ordinal0 [0x008942C7+1131207]
GetHandleVerifier [0x009970FD+523789]
GetHandleVerifier [0x00996E90+523168]
GetHandleVerifier [0x0099E1E7+552695]
GetHandleVerifier [0x009978FA+525834]
Ordinal0 [0x0088B7FC+1095676]
Ordinal0 [0x00782053+8275]
Ordinal0 [0x00781DA7+7591]
GetHandleVerifier [0x00CFA3BC+4075724]
BaseThreadInitThunk [0x74C86359+25]
RtlGetAppContainerNamedObjectPath [0x76F07B74+228]
RtlGetAppContainerNamedObjectPath [0x76F07B44+180]
, error=invalid session id, message=invalid session id}
Build info: version: '3.1.0', revision: 'b94c902', time: '2017-02-16 12:21:31 -0800'
System info: host: 'MININT-SMKHDAE', ip: '192.168.58.87', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_221'
Driver info: driver.version: RemoteWebDriver
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:375)
at org.openqa.selenium.remote.RemoteWebDriver.findElementById(RemoteWebDriver.java:417)
at org.openqa.selenium.By$ById.findElement(By.java:218)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:360)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at com.sun.proxy.$Proxy9.sendKeys(Unknown Source)
at com.learnautomation.pages.LoginPage.loginToHRM(LoginPage.java:26)
at com.learnautomation.testcases.LoginTestHRM.loginToApp(LoginTestHRM.java:22)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.ClassCastException: com.google.common.collect.Maps$TransformedEntriesMap cannot be cast to org.openqa.selenium.WebElement
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:373)
... 33 more
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.WebDriverException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginToApp -->
        <test-method status="FAIL" signature="tearDownMethod(org.testng.ITestResult)[pri:0, instance:com.learnautomation.testcases.LoginTestHRM@291ae]" name="tearDownMethod" is-config="true" duration-ms="6" started-at="2020-02-12T14:34:52Z" finished-at="2020-02-12T14:34:52Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=loginToApp status=FAILURE method=LoginTestHRM.loginToApp()[pri:1, instance:com.learnautomation.testcases.LoginTestHRM@291ae] output={null}]]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[Unexpected result for screenshot command: com.google.common.collect.Maps$TransformedEntriesMap instance]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: Unexpected result for screenshot command: com.google.common.collect.Maps$TransformedEntriesMap instance
at org.openqa.selenium.remote.RemoteWebDriver.getScreenshotAs(RemoteWebDriver.java:349)
at com.learnautomation.utility.Helper.captureScreenshot(Helper.java:20)
at com.learnautomation.pages.BaseClass.tearDownMethod(BaseClass.java:61)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method status="SKIP" signature="tearDown()[pri:0, instance:com.learnautomation.testcases.LoginTestHRM@291ae]" name="tearDown" is-config="true" duration-ms="0" started-at="2020-02-12T14:34:52Z" finished-at="2020-02-12T14:34:52Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.learnautomation.testcases.LoginTestHRM -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
